dotcom-rendering/src/web/components/KeyEventsContainer.test.tsx

```diff
@@ -0,0 +1,82 @@
+import { render } from '@testing-library/react';
+import { ArticleDisplay, ArticleDesign, ArticlePillar } from '@guardian/libs';
+
+import { KeyEventsContainer } from './KeyEventsContainer';
+
+const baseProperties = {
+	id: '123',
+	elements: [],
+	primaryDateLine: '',
+	secondaryDateLine: '',
+};
+
+describe('KeyEventsContainer', () => {
+	it.skip('It should render KeyEventsContainer as expected', () => {
```

Comment:
Should all of these be skipped?

Reply:
We skipped the test because of a jest config issue but that's fixed now